lightkurve/targetpixelfile.py

```diff
@@ -375,6 +399,42 @@ def to_lightcurve(self, method='aperture', **kwargs):
         else:
             raise ValueError("Photometry method must be 'aperture' or 'prf'.")
 
+    def extract_aperture_photometry(self, aperture_mask='pipeline', centroid_method='moments'):
```

Comment:
Just as a heads up: I think I moved `extract_aperture_photometry` to the base class in #651 last week to resolve a bug, so this may trigger a complicated merge conflict.  Apologies!